.travis.yml

```diff
@@ -2,7 +2,7 @@ language: java
 sudo: false
 jdk:
 - oraclejdk8
-- openjdk7
+- openjdk8
```

Comment:
@thinkingserious do we need to support the jdk7? That was what was causing the issue here.